swagger/gateway-swagger.yml

```diff
@@ -33,7 +30,7 @@ info:
     name: Mozilla Public License Version 2.0
     url: https://github.com/TykTechnologies/tyk/blob/master/LICENSE.md
   title: Tyk Gateway API
-  version: 5.10.1
+  version: 5.8.9
```

Comment:
Ah.... edits are being made to reflect 5.8 branch... isn't there a 5.8 version to work from?